src/views/Profile.vue

```diff
@@ -1,13 +1,268 @@
+                  {{ getCreatedAt }}
+                </span>
+              </div>
+              <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:border-b sm:border-gray-200">
+                <span class="text-sm font-medium text-gray-500">
+                  {{ $t('profile.updatedAt.text') }}
+                </span>
+                <span class="flex-grow mt-1 ml-auto flex text-sm text-gray-900 sm:mt-0 sm:col-span-2">
+                  {{ getUpdatedAt }}
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <transition name="fade">
+      <upload-avatar v-if="showModal" @closeModal="showModal = false" />
+    </transition>
+  </div>
 </template>
 
 <script>
+import {mapActions, mapGetters} from 'vuex';
+import userService from '@/api/userService';
+import get from 'lodash/get';
+import format from 'date-fns/format';
+import formatDistance from 'date-fns/formatDistance'
+import UploadAvatar from '@/components/UploadAvatar';
+
 export default {
-  name: "Profile"
+  name: "Profile",
+
+  components: {
+    UploadAvatar
+  },
+
+  data() {
+    return {
+      edit: false,
+      profile: {
+        first_name: '',
+        last_name: ''
+      },
+      blogs: [],
+      comments: [],
+      showModal: false
+    }
+  },
+
+  computed: {
+    ...mapGetters({
+      user: 'userState'
+    }),
+
+    getCreatedAt() {
+      const date = this.user.created_at.substring(0, 10).split('-');
+
+      return date[0]
+          ? format(new Date(date), 'dd MMM yyyy')
+          : "";
+    },
+
+    getUpdatedAt() {
+      const date = this.user.updated_at.substring(0, 10).split('-');
+
+      return date[0]
+          ? formatDistance(
+              new Date(date),
+              new Date(),
+              { addSuffix: true }
+          )
+          : ""
+    }
+  },
+
+  methods: {
+    ...mapActions(['setUserState', 'setProfileState']),
+
+    get, format, formatDistance,
+
+    async onSave() {
+      try {
+        if (!this.profile.first_name || !this.profile.last_name) {
+          this.edit = false;
+
+          this.profile.first_name = this.user.first_name;
+          this.profile.last_name = this.user.last_name;
+
+          throw new Error(this.$t('notifyMessage.error.requiredNames'));
+        }
+
+        await userService.updateProfile(this.profile);
+        const profile = await userService.getProfile();
+
+        let tempProfile = {};
+        tempProfile.first_name = get(
+            profile,
+            'data.attributes.first_name',
+            this.user.first_name
+        );
+        tempProfile.last_name = get(
+            profile,
+            'data.attributes.last_name',
+            this.user.last_name
+        );
+        tempProfile.avatar = get(
+            profile,
+            'data.attributes.avatar',
+            'https://t4.ftcdn.net/jpg/03/46/93/61/360_F_346936114_RaxE6OQogebgAWTalE1myseY1Hbb5qPM.jpg'
```

Comment:
Same as above regarding avatar.
If we need to send this to the API, would suggest at least saving this into a constant `DEFAULT_AVATAR`